app/components-test-lib/page.tsx

```diff
@@ -37,14 +35,9 @@ const ComponentsTestLib = () => {
       <VStack spacing={6} align="start">
         <HStack w="100%">
           <Box w="400px">
-            <SearchBar
-              coordinates={[0, 0]}
-              onSearchChange={() => {}}
-              onAddressSearch={() => {}}
-              onCoordDataRetrieve={() => {}}
-              onHazardDataLoading={() => {}}
-              onSearchComplete={() => {}}
-            />
+            <Suspense>
+              <SearchBarClientWrapper />
+            </Suspense>
```

Comment:
added to satisfy linter not allowing passing of event handlers to client components (see: https://sentry.io/answers/next-js-13-button-onclick-event-handlers-cannot-be-passed-to-client-component-props/)